cmd/wasm-client/go.mod

```diff
@@ -203,8 +203,6 @@ require (
 	sigs.k8s.io/yaml v1.2.0 // indirect
 )
 
-replace github.com/lightninglabs/lightning-node-connect => ../../
-
```

Comment:
Ops, missed including the comment:

Hmmm what's the reasoning behind this change? The reason we had this previously, is that we want the `lightning-node-connect` dependency in the wasm client + mobile, be the latest tagged version during a release of lightning-node-connect. Now the issue is that we can't reference the tagged version in those `go.mod` files, prior to the tagged commit actually existing, which is why we use this replace instead.

I think it's important that we maintain that behaviour, so unless this is a temporary change that we'll fix before the next release, I think we need to address this somehow.

Reply:
This is to temporarily resolve a circular dependency issue we have between litd and LNC. We need to bump the `lnd` version in LNC so we can bump it in `litd`. But because these two submodules reference `litd`, we can't do that.
So as soon as we've updated `litd`, we can revert this again.

Reply:
Ok, thanks for the clarification :pray:!

Reply:
Instead of deleting this replace, it could be good to comment it out and with a `TODO:` and explain what @guggero, just described above. Just that we don't forget readding it later.